conjureup/controllers/newcloud/gui.py

```diff
@@ -132,6 +133,14 @@ def finish(self, credentials=None, back=False):
         if self.cloud == 'maas':
             self.cloud = '{}/{}'.format(self.cloud,
                                         credentials['@maas-server'].value)
+            # Store the MAAS information for the api client
+            maas_api_key = common.parse_maas_apikey(
+                credentials['maas-oauth'].value)
+            app.maas.client = MaasClient(
```

Comment:
OK, nice. I'll actually need it over in deploy/gui instead, since we don't always run the newcloud controller, but I'll do that in a separate branch